lua/plenary/async/util.lua

```diff
@@ -43,9 +43,12 @@ M.block_on = function(async_function, timeout)
   return unpack(ret)
 end
 
-M.will_block = function(async_func)
+---@see M.block_on
```

Comment:
I never really figured out how these annotations work. According to EmmyLua, it is a valid annotation.

Reply:
Whats wrong here?

Just saying but tree-sitter-lua docgen doesnt recognize that as emmy lua because it needs to have a `--- asdf` general description before any annotations.

personally i like `@param` above `@see`.

What isn't working here?

Reply:
I wasn’t sure if the `@see` annotation was being used properly here. Figured it’d make sense to refer to it here since this is just a wrapper around it